pkg/networkdevice/profile/profiledefinition/metrics.go

```diff
@@ -39,6 +39,18 @@ const (
 	ProfileMetricTypePercent ProfileMetricType = "percent"
 )
 
+// SymbolConfigCompat is used to deserialize string field or SymbolConfig.
+// For OID/Name to Symbol harmonization:
+// When users declare metric tag like:
+//
+//	metric_tags:
+//	  - OID: 1.2.3
+//	    name: aSymbol
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏ should be `symbol: aSymbol`
Which explains why it can collide with the new
```
symbol:
   OID: ...
   name: ...
```
syntax if I understand correctly

Reply:
indeed, it's typo, fixed here https://github.com/DataDog/datadog-agent/pull/19925/commits/749922bee7f7ab11777ca7a16aeb066bd5af1e5b